src/Dfe.PlanTech.Web/Controllers/QuestionsController.cs

```diff
@@ -108,17 +107,17 @@ public async Task<IActionResult> SubmitAnswer(SubmitAnswerDto submitAnswerDto, [
         }
 
         string? nextQuestionId;
-        
+
         try
         {
             nextQuestionId = await submitAnswerCommand.GetNextQuestionId(submitAnswerDto.QuestionId, submitAnswerDto.ChosenAnswerId);
         }
         catch (Exception e)
         {
-            logger.LogError( "An error has occurred while retrieving the next question with the following message: {} ", e.Message);
+            logger.LogError("An error has occurred while retrieving the next question with the following message: {} ", e.Message);
```

Comment:
I realise this appears to just be a "fix formatting" edit here, but should there be some name within the braces? Will it work as is?

Reply:
Forced it to throw the error and it _does_ work, _but_ I think it's best practise to name the string interpolations so I quickly added a name anyhow: [735fbfb](https://github.com/DFE-Digital/plan-technology-for-your-school/pull/288/commits/735fbfb778209af04b97b9281e8adc246d82a426)